buffers-and-streams/buffers.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: default
+title: Buffers in depth
+url: buffers
+---
+
+## Buffers
+
+### Usage
+
+The NodeJS Buffer class is a global class and therefore does not need to be imported into a module.
```

Comment:
I've been trying to use `ClassName` formatting for classes - makes it look more visually appealing. Minor detail, but would help.

Reply:
with `Buffer`? sounds good, updated to that but can change

Also renamed to Node, from NodeJS